dotCMS/src/main/java/com/dotmarketing/business/RelationshipAPIImpl.java

```diff
@@ -1,14 +1,200 @@
+        return super.relatedContentTrees(relationship, contentlet, hasParent);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Contentlet> dbRelatedContent(Relationship relationship, Contentlet contentlet, boolean hasParent, boolean live, String orderBy) throws DotDataException {
+        return super.dbRelatedContent(relationship, contentlet, hasParent, live, orderBy);
+    }
+
+    @Override
+    public boolean isParent(Relationship rel, ContentTypeIf st) {
+        return super.isParent(rel, st);
+    }
+
+    @Override
+    public boolean isChild(Relationship rel, ContentTypeIf st) {
+        return super.isChild(rel, st);
+    }
+
+    @Override
+    public boolean sameParentAndChild(Relationship rel) {
+        return super.sameParentAndChild(rel);
+    }
+
+    @WrapInTransaction
+    @Override
+    public void save(Relationship relationship) throws DotHibernateException {
+        super.save(relationship);
+    }
+
+    @WrapInTransaction
+    @Override
+    public void save(Relationship relationship, String inode) throws DotHibernateException {
+        super.save(relationship, inode);
+    }
+
+    @WrapInTransaction
+    @Override
+    public void delete(String inode) throws DotHibernateException {
+        super.delete(inode);
+    }
+
+    @WrapInTransaction
+    @Override
+    public void delete(Relationship relationship) throws DotHibernateException {
+        super.delete(relationship);
+    }
+
+    @WrapInTransaction
+    @Override
+    public void deleteKeepTrees(Relationship relationship) throws DotHibernateException {
+        super.deleteKeepTrees(relationship);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Contentlet> dbRelatedContentByParent(String parentInode, String relationType, boolean live, String orderBy) throws DotDataException {
+        return super.dbRelatedContentByParent(parentInode, relationType, live, orderBy);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public int maxSortOrder(String parentInode, String relationType) {
+        return super.maxSortOrder(parentInode, relationType);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Contentlet> dbRelatedContentByChild(String childInode, String relationType, boolean live, String orderBy) throws DotDataException {
+        return super.dbRelatedContentByChild(childInode, relationType, live, orderBy);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Contentlet> dbRelatedContent(Relationship relationship, Contentlet contentlet, boolean hasParent, boolean live) throws DotDataException {
+        return super.dbRelatedContent(relationship, contentlet, hasParent, live);
+    }
 
+    @CloseDBIfOpened
+    @Override
+    public List<Contentlet> dbRelatedContent(Relationship relationship, Contentlet contentlet, String orderBy, String sqlCondition, boolean liveContent) throws DotDataException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 178 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)